docs/contracts/controller/Reputation.md

```diff
@@ -1,2 +1,17 @@
-A *Reputation* is a way of assigning importance to participants and their votes. 
-*Reputation* is similar to regular tokens but with one crucial difference: **It is non-transferable**.
\ No newline at end of file
+## Reputation
+
+
+A *Reputation* is a way of assigning importance to participants and their votes.
```

Comment:
Maybe influence instead of importance?

Reply:
@leviadam how about : 
> Each DAO has Reputation System which allows peers to rate other peers in order to build trust . A reputation is use to assign influence measure to a DAO'S peers.
For example : A DAO might choose to use a reputation based voting mechainsim in order to take decisions. In this case a peer with relatively higher reputation value will have more influence in the organization.